projects/wonka-ui/package-lock.json

```diff
@@ -4,6 +4,14 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@angular/material": {
```

Comment:
If @angular/material is not a dependency of wonka-ui anymore, this should not be here.

Reply:
Fixed :-)